### ISSUE TITLE
[Issue]: port 8096 binding to all interfaces, not just localhost

### ISSUE THREAD

**Issue by Wurzelmann** [jellyfin/jellyfin#7142]
### Please describe your bug

I'm using Debian without systemd (svsinit instead) and am unable to get jellyfin to bind its port 8096 to localhost only (because I'm using nginx as a reverse proxy).  It always binds at "0.0.0.0:8096", no matter what I enter at the networking section in the GUI.

Is there an option to start jellyfin with local ports only?

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Debian 11.2
- Virtualization: PV (xen)
- Clients: Browser, Android
- Browser: Firefox (91.40esr)
- FFmpeg Version: 4.4.1-1
- Playback Method: Direct Stream
- Hardware Acceleration: none
- Plugins: none
- Reverse Proxy: nginx
- Base URL: none
- Networking: Bridge
- Storage: Samba shares
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
The settings for binding to specific network addresses is broken in 10.7.x

It got fixed and works as intended on 10.8 unstable.

**Comment by Wurzelmann** [comment]
Oh sorry, I was not aware. Any idea when it will become stable?

**Comment by Wurzelmann** [comment]
bump

**Comment by crobibero** [comment]
As stated, this is fixed in 10.8
